app/graphql/types/query_type.rb

```diff
@@ -6,6 +6,14 @@ class QueryType < Types::BaseObject
       argument :id, ID, required: true
     end
 
+    field :albums, [AlbumType], null: false do
+      description 'List all albums'
+    end
+
+    def albums
+      AlbumsService.new(ENDPOINT).list_albums
```

Comment:
I think you can use the `ENDPOINT` inside AlbumService

Reply:
if i dont initialize AlbumService with the ENDOPOINT parameter, i cant use this variable in this class